Resources/public/css/views/fields/edit/author.css

```diff
@@ -32,3 +32,7 @@
     display: block;
     margin-top: 3em;
 }
+
+.ez-view-authoreditview.is-not-translatable .ez-field-sublabel {
+    opacity: 0.3;
```

Comment:
all the opacity changes (in this CSS file and the others) should be moved in the theme CSS